ACPs/151-icm-use-current-block-pchain-height/README.md

```diff
@@ -0,0 +1,38 @@
+| ACP           | 151                                                             |
+| :------------ | :-------------------------------------------------------------- |
+| **Title**     | Use current block P-Chain height for ICM signature verification |
+| **Author(s)** | Ian Suvak ([@iansuvak](https://github.com/iansuvak))            |
+| **Status**    | Proposed                                                        |
+| **Track**     | Standards                                                       |
+
+## Abstract
+
+Proposes that VMs use the P-Chain height of the current block being built instead of the parent block when verifying aggregated signatures of Avalanche Interchain Messages (ICM). This will allow for a more reliable way to determine which validators should participate in signing the message, and remove unnecessary waiting periods.
+
+## Motivation
+
+Currently ICM messages use P-Chain height of the parent block when verifying messages for inclusion in the current block. Using the parent's P-Chain height is necessary for safe block building purposes but not for ICM message verification. 
```

Comment:
```suggestion
Currently the ProposerVM uses the P-Chain height of the parent block when verifying ICM messages in the current block. Using the parent block's P-Chain height is necessary for verifying the proposer and reaching consensus on the current block, but it is not necessary not for verifying ICM messages within the block.
```

Reply:
I like the second part of the suggestion but I have the same comment as above, re: wording implying that the ProposerVM does ICM message verification. My current wording is worse since it implies that ICM messages verify themselves 😓 . 

Should I just have the proposal / title of the ACP be `Use current block's P-Chain height for block state verification`.

And then mention ICM as the existing use-case in the motivation.

Reply:
> Should I just have the proposal / title of the ACP be Use current block's P-Chain height for block state verification. And then mention ICM as the existing use-case in the motivation.

I hadn't thought of that, but yeah it makes sense to me. Inner VMs could use this P-Chain for other purposes in the future hypothetically. 

I also updated the suggested wording here to reflect what you called out. Great point